CONTRIBUTING.md

```diff
@@ -43,6 +43,34 @@ If any of aforementioned labels don't have unassigned issues, please [contact] o
 [maintainers]: https://github.com/etcd-io/etcd/blob/main/OWNERS
 [priority/important]: https://github.com/search?type=issues&q=org%3Aetcd-io+state%3Aopen++label%3A%22priority%2Fimportant%22
 
+### Flaky Tests
```

Comment:
pls add an entry in the table of contents.

Reply:
done